<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-OND3CLR" failures="0" tests="2" name="TestRunner" time="0.998" errors="1" timestamp="3 Feb 2020 15:21:42 GMT">
  <testcase classname="TestRunner" name="modificationAdresse" time="0.001">
    <error message="Unresolved compilation problems: 
	The method updateAdresse(boolean, String, String, String, int) in the type IAdresseDao is not applicable for the arguments (Adresse)
	The method updateAdresse(boolean, String, String, String, int) in the type IAdresseDao is not applicable for the arguments (Adresse)
	Type mismatch: cannot convert from int to Adresse
	Type mismatch: cannot convert from int to Adresse
" type="java.lang.Error">
      <![CDATA[java.lang.Error: Unresolved compilation problems: 
	The method updateAdresse(boolean, String, String, String, int) in the type IAdresseDao is not applicable for the arguments (Adresse)
	The method updateAdresse(boolean, String, String, String, int) in the type IAdresseDao is not applicable for the arguments (Adresse)
	Type mismatch: cannot convert from int to Adresse
	Type mismatch: cannot convert from int to Adresse

	at TestRunner.modificationAdresse(TestRunner.java:94)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- modificationAdresse -->
  <testcase classname="TestRunner" name="feature" time="0.997"/>
</testsuite> <!-- TestRunner -->
